StringCompression.py

```diff
@@ -0,0 +1,24 @@
+def stringCompression(s):
+    if not s:
+        return s
+    comp = []
+    cur = s[0]
+    count = 1
+    for i in range(1, len(s)):
+        if s[i] == cur:
+            count += 1
+        else:
+            comp.append(cur)
+            comp.append(str(count))
```

Comment:
This could be in one line.
```suggestion
            comp.append(cur + str(count))
```